packages/qcheck-alcotest/qcheck-alcotest.0.9/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes <simon.cruanes.2007@m4x.orgr>"
```

Comment:
should this be @m4x.org? Similar problem in the other opam files.

Reply:
well spotted -_-
I'll fix in my repo, that's fine anyway since the real address is just above…